### DISCUSSION TITLE
🔗 SendFox integration (Duplicate of #20375)

### DISCUSSION THREAD

**Discussion by donwinchell** [elementor/elementor#20409] [Feature Request]
I am having a tought time figureing out this request meathod.  I found a discousssion with at least 12 people wanting SendFox integration but I am the only one here,  in any case this is a request and here is some information from SendFox on how to do it.
https://help.sendfox.com/article/276-sendfox-integrations

**Comment by nicholaszein** [comment]
Hello @donwinchell!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #20375

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
